monero-outreach-docs/translations/es/PSA/files/learning/learning_es.sla

```diff
@@ -64,15 +68,59 @@
-            <trail LINESP="13.2"/>
+        <PAGEOBJECT OwnPage="0" PTYPE="4" XPOS="424" YPOS="380" WIDTH="252" HEIGHT="190" RADRECT="0" FRTYPE="0" CLIPEDIT="0" PWIDTH="1" PCOLOR="None" PCOLOR2="None" TXTFILLSH="90" COLUMNS="1" COLGAP="0" NAMEDLST="" SHADE="100" SHADE2="100" GRTYP="0" ROT="0" PLINEART="1" PLINEEND="0" PLINEJOIN="0" LINESP="13.2" LOCALSCX="1" LOCALSCY="1" LOCALX="0" LOCALY="0" PICART="1" PLTSHOW="0" BASEOF="0" textPathType="0" textPathFlipped="0" FLIPPEDH="0" FLIPPEDV="0" IFONT="Adobe Caslon Pro Regular" ISIZE="11" SCALETYPE="1" RATIO="1" PRINTABLE="1" ANNOTATION="0" ANNAME="" TEXTFLOWMODE="0" TEXTFLOW="0" TEXTFLOW2="0" TEXTFLOW3="0" AUTOTEXT="0" EXTRA="0" TEXTRA="0" BEXTRA="0" REXTRA="0" FLOP="0" PFILE="" PFILE2="" PFILE3="" PRFILE="" EPROF="" IRENDER="1" EMBEDDED="1" LOCK="1" LOCKR="0" REVERS="0" TransValue="0" TransValueS="0" TransBlend="0" TransBlendS="0" isTableItem="0" TopLine="0" LeftLine="0" RightLine="0" BottomLine="0" isGroupControl="0" NUMDASH="0" DASHS="" DASHOFF="0" NUMPO="16" POCOOR="0 0 0 0 252 0 252 0 252 0 252 0 252 190 252 190 252 190 252 190 0 190 0 190 0 190 0 190 0 0 0 0 " NUMCO="16" COCOOR="0 0 0 0 252 0 252 0 252 0 252 0 252 190 252 190 252 190 252 190 0 190 0 190 0 190 0 190 0 0 0 0 " NUMGROUP="0" GROUPS="" startArrowIndex="0" endArrowIndex="0" OnMasterPage="" ImageClip="" ImageRes="1" Pagenumber="0" isInline="0" fillRule="1" doOverprint="0" gXpos="0" gYpos="0" gWidth="0" gHeight="0" LAYER="0" BOOKMARK="0" NEXTITEM="-1" BACKITEM="-1">
+            <ITEXT FSHADE="90" CH="Juntos podemos fortalecer el ecosistema de Monero aprendiendo más sobre sus características, el código, "/>
+            <para LINESP="13.2"/>
+            <ITEXT FSHADE="90" CH="sus valores fundados sobre la cultura ciberpunk de código abierto y las maneras de mejorar la privacidad y seguridad al utilizarlo. Al tener una mayor comprensión de los aspectos de Monero, tendrás la habilidad de hablarle a otros y ayudar a que haya una mayor adopción. Sentirás mayor confianza y seguridad al guardar, usar y hacer transacciones con XMR. El conocimiento es poder y puede cambiar el mundo. Siempre busca aprender más…y ¡has tus propias investigaciones!"/>
```

Comment:
"Siempre busca aprender más… **y** (missing space after … and a comma after "y"), ¡**haz** tus propias investigaciones!". The _Haz_ used here is from the Spanish verb "hacer" (do), not from the verb "haber" (to have) :).

Reply:
right, done